project-koku/koku
# Fixing non trino effective cost

Fixes [COST-2217]

Changes proposed in this PR:
* Fixing effective cost usage for non-trino flows

Testing Instructions:
1. Start in non trino mode
2. add ocp source
3. create cost model with effective cost metrics
4. check cost show on openshift costs endpoint.
